### DISCUSSION TITLE
Sandbox errors, but boots when forked

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#5083] [Q&A]
https://codesandbox.io/embed/github/NullVoxPopuli/ember-new-output/tree/stable/?codemirror=1

This errors. But when forked, starts up just fine.

What's wrong with the original?

**Comment by lbogdan** [comment]
Hey @NullVoxPopuli ,

There was an issue with the persistent storage for that sandbox, should be fixed now. Sorry for the inconveniences, please let us know it's working on your side.

**Reply by NullVoxPopuli** [reply]
Thanks! Works fine now!
